theano/gpuarray/opt.py

```diff
@@ -82,6 +106,15 @@ def f(local_opt):
     return f
 
 
+def register_opt2(tracks, *tags, **kwargs):
```

Comment:
There should be documentation here explaining what `local_opt` should look like: which arguments, what does it return, etc.
It does not have the same signature as the other `local_...` functions that are passed to `FromFunctionLocalOptimizer`, so it should be documented.